### PR TITLE
Added Translator

### PR DESCRIPTION
**Pull request by DoYouNotice**
## Arna Translate
- [x] Installed and tested by me on my main smartphone and used for about 15 minutes or more.
- [x] Has dark theme.
- [ ] It has x trackers on [εxodus](εxodus report link).
- [x] It has 1 F-Droid [Anti-Features](https://apt.izzysoft.de/fdroid/index/apk/com.arna.translate).
- [ ] I use the app on a daily basis and I love almost the entirety of it, so I placed next to it a heart.

## SimplyTranslate Mobile 
- [x] Installed and tested by me on my main smartphone and used for about 15 minutes or more.
- [x] Has dark theme.
- [x] It has 0 trackers on [εxodus](https://reports.exodus-privacy.eu.org/en/reports/com.simplytranslate_mobile/latest/).
- [x] It has 1 F-Droid [Anti-Features](https://reports.exodus-privacy.eu.org/en/reports/com.simplytranslate_mobile/latest/).
- [x] I use the app on a daily basis and I love almost the entirety of it, so I placed next to it a heart.

### CODE REVIEW

**Review by XXDRD — APPROVED** [review]
Nice apps

**Comment by XXDRD** [comment]
Anyway, you don't have to add the app in the categories with an alphabetical order. This makes addition of apps by new contributors easier

File: README.md [hunk 1]
```diff
@@ -1466,6 +1467,24 @@ Currently being reworked as Neo Launcher.
...
+- [x] [GitHub](https://github.com/MahanRahmati/translate)
+- [x] [Official page](https://lingva.ml/)
+
+### SimplyTranslate Mobile ❤️
+<img alt="SimplyTranslate Mobile" height="64" src="https://github.com/ManeraKai/simplytranslate_mobile/raw/main/fastlane/metadata/android/en-US/images/icon.png">
+
+- [ ] Google Play
+- [x] [F-Droid](https://f-droid.org/en/packages/com.simplytranslate_mobile/)
+- [x] [GitHub](https://github.com/ManeraKai/simplytranslate_mobile)
+- [x] [Official page](https://simplytranslate.org/)
```
Comment (Psyhackological) [inline README.md:1486]: Should not be this site? https://manerakai.github.io/simplytranslate_mobile/

Comment (DoYouNotice) [reply README.md:1486]: yes i choose to use the instance instead, should i remove it. also did that for arna translate

**Comment by Psyhackological** [comment]
Hey @DoYouNotice. Could you close your pull requests and make a single one with all those changes that you've proposed? There are some merge conflicts which I can resolve, but I would be pain to work with and it can be done easier. Also I can just add everything by myself, however to appreciate your contribution so I want to merge your changes.

**Comment by DoYouNotice** [comment]
> Hey @DoYouNotice. Could you close your pull requests and make a single one with all those changes that you've proposed? There are some merge conflicts which I can resolve, but I would be pain to work with and it can be done easier. Also I can just add everything by myself, however to appreciate your contribution so I want to merge your changes.

sure

**Pull request closed without merging**